src/overlays/actors/ovl_En_Scopecoin/z_en_scopecoin.c

```diff
@@ -21,16 +23,82 @@ const ActorInit En_Scopecoin_InitVars = {
     (ActorFunc)EnScopecoin_Update,
     (ActorFunc)EnScopecoin_Draw
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Scopecoin_0x80BFCFA0/func_80BFCFA0.asm")
+extern UNK_PTR D_04061FC0[];
+extern UNK_PTR D_04061FE0[];
+extern UNK_PTR D_04062000[];
+extern UNK_PTR D_04062040[];
+extern UNK_PTR D_04062020[];
+extern UNK_PTR D_04062060[];
+extern UNK_PTR D_04062000[];
+extern Gfx D_040622C0[];
```

Comment:
These symbols, since they are from `gameplay_keep` and not from this actor per se, should be placed in `include\segment.h`